### DISCUSSION TITLE
请问，你笔记中提到的一篇论文，你有他的源码吗？

### DISCUSSION THREAD

**Discussion by GOODirty** [openwhu/OpenWHU#128] [Q&A]
Quantization Index Modulation: A Class of Provably
Good Methods for Digital Watermarking and
Information Embedding
Brian Chen, Member, IEEE, and Gregory W. Wornell, Senior Member, IEEE这篇论文

**Comment by Crescentm** [comment]
QIM谷歌能搜索到很多实现，关键词可以是"Quantization Index Modulation Github"。
以及提问的时候，最好把关键信息放在标题里。
